src/index.html

```diff
@@ -12,7 +12,39 @@
       href="styles/index.scss"
```

Comment:
You should link a CSS file instead of SCSS. Browsers do not understand SCSS syntax, so you must compile it to CSS and link the compiled file.